content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -288,13 +288,8 @@ You can install only one pod network per cluster.
 
 {{< tabs name="tabs-pod-install" >}}
 {{% tab name="Choose one..." %}}
```

Comment:
Aside: I'm not sure I'd keep the “Choose one...” tab. That can come out in a different pull request.

Reply:
removed and extracted one sentence from it.